Makefile

```diff
@@ -16,9 +16,10 @@ release-darwin-%:
 	cp artifacts/darwin-$*/libtokenizers.darwin-$*.tar.gz artifacts/all/libtokenizers.darwin-$*.tar.gz
 
 release-linux-%:
-	docker buildx build --platform linux/$* -f release/Dockerfile . -t tokenizers.linux-$*
+	docker buildx build --platform linux/$* --build-arg="DOCKER_TARGETPLATFORM=linux/$*" -f release/Dockerfile . -t tokenizers.linux-$*
```

Comment:
This extra build arg is dumb.  However, Rust, docker, and golang accept many different OS/ARCH combination names and they all "default" to one name or another.  This does not let us sync up the file paths very easily.  The easiest solution was just accept an extra param so we are very explicit on which platform we are wanting to use